### ISSUE TITLE
1.0.4 version failed Kubernetes API

### ISSUE THREAD

**Issue by jmorcar** [influxdata/helm-charts#57]
I checked an error in Kubernetes input plugin when I deployed 1.0.4 version:
Error in plugin: https://10.200.0.35:10250/pods returned HTTP status 403 Forbidden

Commands to reproduce, my helm chart had 1.0.3 version:

```
$ helm upgrade -f ./config.yaml --force telegraf-ds influxdata/telegraf-ds
Release "telegraf-ds" has been upgraded. Happy Helming!
NAME: telegraf-ds
LAST DEPLOYED: Tue Apr  7 13:38:48 2020
NAMESPACE: logstreaming
STATUS: deployed
REVISION: 5
TEST SUITE: None
NOTES:
[..]


$ kubectl logs telegraf-ds-ggzhn
2020-04-07T11:38:58Z I! Starting Telegraf 1.14.0
2020-04-07T11:38:58Z I! Using config file: /etc/telegraf/telegraf.conf
2020-04-07T11:38:58Z I! Loaded inputs: disk kubernetes diskio kernel swap cpu mem processes system docker
2020-04-07T11:38:58Z I! Loaded aggregators: 
2020-04-07T11:38:58Z I! Loaded processors: 
2020-04-07T11:38:58Z I! Loaded outputs: influxdb
2020-04-07T11:38:58Z I! Tags enabled: host=aks-otherpurpose-28286210-vmss000000
2020-04-07T11:38:58Z I! [agent] Config: Interval:10s, Quiet:false, Hostname:"aks-otherpurpose-28286210-vmss000000", Flush Interval:10s
2020-04-07T11:39:00Z E! [inputs.kubernetes] Error in plugin: https://10.200.0.35:10250/pods returned HTTP status 403 Forbidden
2020-04-07T11:39:10Z E! [inputs.kubernetes] Error in plugin: https://10.200.0.35:10250/pods returned HTTP status 403 Forbidden
```
But If upgrade to 1.0.3 revision again , is OK:

```
helm upgrade -f ./config.yaml --force telegraf-ds influxdata/telegraf-ds --version 1.0.3
Release "telegraf-ds" has been upgraded. Happy Helming!
NAME: telegraf-ds
LAST DEPLOYED: Tue Apr  7 13:39:45 2020
NAMESPACE: logstreaming
STATUS: deployed
REVISION: 6
TEST SUITE: None
NOTES:
[....]


$ kubectl logs telegraf-ds-2l5k2
2020-04-07T11:39:58Z I! Starting Telegraf 1.13.4
2020-04-07T11:39:58Z I! Using config file: /etc/telegraf/telegraf.conf
2020-04-07T11:39:58Z I! Loaded inputs: diskio kernel processes swap disk docker mem system cpu kubernetes
2020-04-07T11:39:58Z I! Loaded aggregators: 
2020-04-07T11:39:58Z I! Loaded processors: 
2020-04-07T11:39:58Z I! Loaded outputs: influxdb
2020-04-07T11:39:58Z I! Tags enabled: host=aks-otherpurpose-28286210-vmss000000
2020-04-07T11:39:58Z I! [agent] Config: Interval:10s, Quiet:false, Hostname:"aks-otherpurpose-28286210-vmss000000", Flush Interval:10s
```
Nothing errors with 1.0.3

**Comment by rawkode** [comment]
Hi @jmorcar,

Thanks for the report. The only change was the bump from Telegraf 1.13 to 1.14 - please bare with me while I run some tests and confirm that this is a regression in Telegraf.

If so, we'll revert the image we use until it's fixed.

**Comment by rawkode** [comment]
Sorry for the delay, @jmorcar . I've confirmed this is a regression in Telegraf 1.14 and I am working on a fix

Please update your `values.yaml` to override the `tag` to `1.13-alpine`

**Comment by rawkode** [comment]
@jmorcar PR open with RBAC fix for the new Kubernetes API calls in 1.14
